.circleci/config.yml

```diff
@@ -116,20 +116,20 @@ jobs:
           name: Lint styles
           command: npx stylelint -- --fix
 
-  # test:
-  #   <<: *defaults
+  test:
```

Comment:
@evexoio We actually use `npm` in Circle, so can you please remove lines 126-128 and change line 132 to `command: npm test`?

Reply:
Sure